spec/compiler/crystal/tools/context_spec.cr

```diff
@@ -1,26 +1,24 @@
 require "spec"
 require "yaml"
-require "../../../../src/compiler/crystal/**"
```

Comment:
@bcardiff as far as i'm concerned we shouldn't be supporting not using `bin/crystal`. All the specs for the stdlib use non-relative requires, so I don't see why the compiler should be special.